keyboards/ymdk/oni_tsangan/readme.md

```diff
@@ -0,0 +1,19 @@
+# YMDK ONI Tsangan
+
+![ONI Tsangan PCB, Hotswap Tsangan/HHKB-like PCB with RGB Matrix and backlight](https://imgur.com/DB87on8.jpg)
+
+Customizable 60% Tsangan/HHKB-like Hotswap PCB by [YMDK](https://ymdkey.com).
+
+- Keyboard Maintainer: [chrisgve](https://github.com/chrisgve)
+- Hardware Supported: YMDK ONI Tsangan/HHKB-like
+- Hardware Availability: [YMDK ONI Tsangan/HHKB-like](https://ymdkey.com/products/ymdk-gh60-hhkb-style-tsangan-oni-hot-swappable-both-underglow-switch-rgb-via-programmable-pcb-compatible-with-tokyo-60)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ymdk/oni_tsangan:default
+
+The keyboard uses a DFU bootloader. To make a keymap and use dfu to flash it:
+
+    make ymdk/oni_tsangan:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
From the PR checklist
> clear instructions on how to reset the board into bootloader mode

Can we include the bootloader section from the template <https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md#bootloader> (updated to correctly reflect the pcb).